packer_files/vsphere_centos_template/centos-config/centos8-ks.cfg

```diff
@@ -0,0 +1,53 @@
+#version=RHEL8
+# Use graphical install
+graphical
+
+%packages
+@^server-product-environment
+kexec-tools
+open-vm-tools
+openssh-server
+perl
```

Comment:
What do we need `perl` for?

Reply:
You need perl and cloud-init for the terraform clone